packages/__vital__/src/shadow/@bodiless/vital-navigation/MenuTitle.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { vitalMenuTitleBase } from '@bodiless/vital-navigation';
+import { asFluidToken } from '@bodiless/vital-elements';
+import { addProps } from '@bodiless/fclasses';
+
+const Default = asFluidToken({
+  ...vitalMenuTitleBase.Default,
+  Compose: {
+    ...vitalMenuTitleBase.Default.Compose,
+    _: addProps({ 'data-shadowed-by': '__vital__:MenuTitle' }),
+  },
+});
+
+export default {
+  /* TODO:  works as is but if uncomment line 34 it gives error:
```

Comment:
@wodenx  This didn't make sense why I couldn't spread base on export like the others.